tests/test_linkedlist.py

```diff
@@ -0,0 +1,55 @@
+"""Module to test the LinkedList implementations."""
```

Comment:
I think there is no need for BaseTestLinkedList class since we only have one implementation of the linked list. Maybe add all the tests in the `TestLinkedList` class and create tests for every method available on the linked list.

List of methods:
- add(item: _type_) -> None
- insert(item: _type_, index:int) -> None
- get(index: int) -> _type_
- clear() -> None
- contains(item: _type_) -> bool
- copy_to(list: list(), index: int) -> None
- remove(item: _type_) -> None
- is_empty() -> bool
- length() -> int